Template2018/src/main/java/frc/team281/robot/subsystems/RealDriveSubsystem.java

```diff
@@ -145,22 +168,31 @@ public void tankDrive(double left, double right) {
 		}
 	}
 
+	@Override
+	public void periodic() {
+        // method called by scheduler automatically
+		if (this.navX != null) {
+            SmartDashboard.putData("NavX: ", this.navX);
```

Comment:
please use dataLogger.log() for this. that way, we can also send it other places if we'd like

Reply:
You're right -- I'll fix it.  I guess old habits die hard :-).